src/CMakeLists.txt

```diff
@@ -323,6 +321,21 @@ if(Qt5Gamepad_FOUND)
   message(STATUS "Using Gamepad module")
 endif(Qt5Gamepad_FOUND)
 
+if(USE_OWN_QT5KEYCHAIN)
+  # An extra definition is needed if we are using our own QtKeyChain
+  target_compile_definitions(mudlet PRIVATE INCLUDE_OWN_QT5_KEYCHAIN QTKEYCHAIN_NO_EXPORT)
+  message(STATUS "Using own QtKeyChain (Qt5) library.")
```

Comment:
This message is already output by the `include_optional_module` helper, so I'd skip it here.

Reply:
Fair comment but it does seem like we get two sets of status messages put out and I am not sure that the first batch gets repeated when the build process is repeated, unless the whole CMake configuration of the project gets blown away first. I have another look and see...

Reply:
.. okay it has been removed.